content/md/what-is/what-is-an-attribute.md

```diff
@@ -98,6 +98,15 @@ For instance, this feature will allow you to have one product name or a descript
 
 In this case, when creating your attribute, enable the `value per locale` property.
 
+## The `Attribute description` property
+
+Thanks to this property, it is possible to explain what the expectations are for each attribute, in terms of enrichment. In other word, this description enables to define the enrichment guidelines.
```

Comment:
"In other words, this description allows to define the enrichment guidelines" ou "In other words, this description defines the enrichment guidelines"